### PR TITLE
List documentation

### PR DESCRIPTION
**Pull request by Shubha932000**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/overview/document-generation-api/templatetags.md [hunk 1]
```diff
@@ -284,6 +275,61 @@ over the specified array of objects.
...
+
+Placeholder tag is replaced by a list generated using the html string provided in the input json.
+
+![Sample for HTML Table insertion](../images/Nested_Numbered_Lists_Example.png)
+
+<InlineAlert slots="text"/>
+
+- Placeholder Tags containing `<ul>` or `<ol>` HTML elements as values in input JSON data should be placed in a separate paragraph in the input document template.
+
+- Only Inline Styling of List elements is possible. i.e. using style attribute of `<ol>` & `<ul>`. Internal and External styling are not allowed.
```
Comment (piyuverma) [inline src/pages/overview/document-generation-api/templatetags.md:313]: possible -> allowed.

Remove Internal and External Styling line as suggested by Ray.

**Review by piyuverma — CHANGES_REQUESTED** [review]
Changes Requested

**Comment by piyuverma** [comment]
https://developer-stage.adobe.com/document-services/docs/overview/document-generation-api/stylingformattingtags/#tags-supported

We should add <ul> and <ol> as part of this list as well and add a link to the list section like
List Html Tags (<ul>, <ol>) where List Html Tags will be hyperlinked to the list section we have created.

**Comment by piyuverma** [comment]
Also, we should add a **NOTE** section in https://developer-stage.adobe.com/document-services/docs/overview/document-generation-api/stylingformattingtags/#tags-supported only, stating the npsp limitation.

File: src/pages/overview/document-generation-api/templatetags.md (hunk 1 above)
Comment (Shubha932000) [reply src/pages/overview/document-generation-api/templatetags.md]: Done

**Comment by Shubha932000** [comment]
> Also, we should add a **NOTE** section in https://developer-stage.adobe.com/document-services/docs/overview/document-generation-api/stylingformattingtags/#tags-supported only, stating the npsp limitation.

added - https://github.com/AdobeDocs/pdfservices-api-documentation/pull/808/files#diff-ed0623261f794c73b176df37f09d0b9391a79311ba75ced1f1a7d7dab2c71b83R128

**Comment by Shubha932000** [comment]
> https://developer-stage.adobe.com/document-services/docs/overview/document-generation-api/stylingformattingtags/#tags-supported
> 
> We should add
> 
> and
> as part of this list as well and add a link to the list section like
> List Html Tags (
> ,
> ) where List Html Tags will be hyperlinked to the list section we have created.

Done

**Pull request closed without merging**